lib/voxpupuli/acceptance/spec_helper_acceptance.rb

```diff
@@ -21,9 +21,9 @@ def configure_beaker(modules: :metadata, &block)
       unless %w[none preinstalled].include?(collection)
         BeakerPuppetHelpers::InstallUtils.install_puppet_release_repo_on(host, collection)
       end
-      package_name = ENV.fetch('BEAKER_PUPPET_PACKAGE_NAME',
-                               BeakerPuppetHelpers::InstallUtils.puppet_package_name(host,
-                                                                                     prefer_aio: collection != 'none'))
+      package_name = ENV.fetch('BEAKER_PUPPET_PACKAGE_NAME') do
+        BeakerPuppetHelpers::InstallUtils.collection2packagename(host, collection, prefer_aio: collection != 'none')
```

Comment:
We can probably push the `prefer_aio` logic into the helper. https://github.com/voxpupuli/beaker_puppet_helpers/pull/62 does that.